ui/v2/src/index.scss

```diff
@@ -38,7 +38,7 @@ code {
   flex-flow: row wrap;
   justify-content: center;
   margin: $pt-grid-size $pt-grid-size 0 0;
-  padding: 0 calc(10%);
```

Comment:
Can we make this and line 71 an interface option?

Reply:
For line 71, I've added a zoom slider to adjust the sizes of the scene cards to one of four different sizes. These sizes can be overridden with the custom css if wanted.

For line 41, I think we should just make a reasonable default and allow it to be overridden with custom css.